Don't check if libtool can do static libraries. This doesn't work
because libtool has not been generated by this point in the configure
script and is unnecessary because the value of STATIC doesn't end up
being used anywhere.
--- configure.ac.orig	2003-02-21 21:06:40.000000000 -0600
+++ configure.ac	2022-05-07 03:48:56.000000000 -0500
@@ -117,12 +117,6 @@
 dnl
 AM_PROG_LIBTOOL
 AC_SUBST(LIBTOOL_DEPS)
-if ${CONFIG_SHELL} ./libtool --features | grep "enable static" >/dev/null; then
-  STATIC="-static"
-else
-  STATIC=
-fi
-AC_SUBST(STATIC)
 
 dnl
 dnl Checks for typedefs, structures, and compiler characteristics.
